agendas/2024/12-Dec/05-wg-primary.md

```diff
@@ -154,3 +155,5 @@ hold additional secondary meetings later in the month.
    - [RFC](https://github.com/graphql/graphql-spec/pull/1056) - promote to RFC2? Merge as an editorial fix?
 1. Be explicit about list coercion (15m, Benjie)
    - [RFC](https://github.com/graphql/graphql-spec/pull/1058) - promote to RFC2? Merge as an editorial fix?
+1. Interface field argument default values (@yaacovCR, 30m)
+   - Spec bug? https://github.com/graphql/graphql-spec/issues/1121
```

Comment:
Is it the last item again? I hope we get to it! There was so much teasing last month :)!

Reply:
So much graphql fun :)